chassis.py

```diff
@@ -77,40 +79,59 @@ def set_state(self, command):
             self.velocityDict[motor] = 0
 
         if command:
+            if command == 'Rotate arm':
+                self.state = 'Arm rotating'
+                self.velocityDict['arm'] = 10
+            # wheel
+            if command[:4] == 'Turn' and command[:5] != 'Turn ':
+                shift = float(command[4:]) * 10
+                self.state = command
+                self.velocityDict['wheel1'] = self.defaultVelocity + shift
+                self.velocityDict['wheel2'] = self.defaultVelocity + shift
+                self.velocityDict['wheel3'] = self.defaultVelocity - shift
+                self.velocityDict['wheel4'] = self.defaultVelocity - shift
+
```

Comment:
我之前给你看的截图我是瞎取名的, 这里建议换成更容易理解的变量名.`self.steer_coefficient`在_init_()进行设置. 你不用接受我这个建议更改, 我只是方便你看区别所以用的这种. 他似乎一次接受多个建议的功能还有点问题, 可能得辛苦你自己改下.
```suggestion
                steer = float(command[4:]) * self.steer_coefficient
                self.state = 'Steering speed: ' + str(steer)
                self.velocityDict['wheel1'] = self.defaultVelocity + steer
                self.velocityDict['wheel2'] = self.defaultVelocity + steer
                self.velocityDict['wheel3'] = self.defaultVelocity - steer
                self.velocityDict['wheel4'] = self.defaultVelocity - steer
```